ethereum_history_api/circuits/lib/src/get_header.nr

```diff
@@ -13,52 +14,55 @@ global RECEIPTS_ROOT_INDEX = 5;
 global BLOCK_NUMBER_INDEX = 8;
 
 struct BlockHeaderPartial {
-    state_root: [u8; HASH_LENGTH],
-    transactions_root: [u8; HASH_LENGTH],
-    receipts_root: [u8; HASH_LENGTH],
     number: Field,
-    hash: [u8; HASH_LENGTH],
-    encoded_len: Field,
-    encoded: [u8; MAX_HEADER_RLP_SIZE]
+    hash: Bytes32,
+    state_root: Bytes32,
+    transactions_root: Bytes32,
+    receipts_root: Bytes32,
+}
+
+struct BlockHeaderEncoded {
+    length: Field,
+    data: [u8; MAX_HEADER_RLP_SIZE]
```

Comment:
nice naming 👍

Reply:
Alternatively, I agree with @akonior, that we could name it BlockHeaderRLP

Reply:
Using camel case naming for `RLP` `BlockHeaderRlp` to maintain consistency with our current approach in other parts of code.